executor/common_linux.h

```diff
@@ -1349,6 +1367,7 @@ static void initialize_netdevices(void)
 	// - ifb0/1
 	// - teql0
 	// - eql
+	// Note: netdevsim devices can't have the same name even in different namespaces.
 	char netdevsim[16];
 	sprintf(netdevsim, "netdevsim%d", (int)procid);
```

Comment:
$cat /proc/sys/kernel/pid_max
4194304

strlen("netdevsim4194304") == 16. No space for zero termination.

Reply:
This is not "pid", this is our test process number from 0 .. to small N.